### PR TITLE
feat: Added alarm clock icon

### PR DESCRIPTION
**Pull request by shridmishra**
[Screencast from 05-11-24 11:50:20 AM IST.webm](https://github.com/user-attachments/assets/096a4528-65d0-4074-bada-6e029d292a17)

added a new alarm clock icon which vibrates on hover

### CODE REVIEW

File: icons/alarm-clock.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      x: {
+        duration: 0.3,
+        repeat: Infinity,
+        ease: "linear"
+      }
+    },
+  },
+};
+
+const AlarmClock = () => {
```
Comment (pqoqubbw) [inline icons/alarm-clock.tsx:65]:
```suggestion
const AlarmClockIcon = () => {
```

File: icons/alarm-clock.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+          initial="normal"
+          animate={controls}
+          d="M6 2L2.25304 5.31064"
+        />
+      </svg>
+    </div>
+  );
+};
+
+export { AlarmClock };
```
Comment (pqoqubbw) [inline icons/alarm-clock.tsx:126]:
```suggestion
export { AlarmClockIcon };
```

File: icons/index.ts [hunk 3]
```diff
@@ -67,6 +68,7 @@ const ICON_LIST = [
   { name: 'archive', icon: ArchiveIcon },
   { name: 'copy', icon: CopyIcon },
   { name: 'attach-file', icon: AttachFileIcon },
+  { name: 'alarm-clock', icon: AlarmClock },
```
Comment (pqoqubbw) [inline icons/index.ts:71]:
```suggestion
  { name: 'alarm-clock', icon: AlarmClockIcon },
```

File: icons/index.ts [hunk 4]
```diff
@@ -44,6 +44,7 @@ import { AttachFileIcon } from '@/icons/attach-file';
 import { GaugeIcon } from '@/icons/gauge';
 import { MenuIcon } from '@/icons/menu';
 import { ClockIcon } from '@/icons/clock';
+import { AlarmClock } from '@/icons/alarm-clock';
```
Comment (pqoqubbw) [inline icons/index.ts:47]:
```suggestion
import { AlarmClockIcon } from '@/icons/alarm-clock';
```

Comment (shridmishra) [reply icons/index.ts]: I've made the requested changes. Please review when you have a moment.

Comment (pqoqubbw) [reply icons/index.ts]: thanks, and could you please fix merge conflicts

File: icons/index.ts [hunk 5]
```diff
@@ -44,12 +44,15 @@ import { AttachFileIcon } from '@/icons/attach-file';
 import { GaugeIcon } from '@/icons/gauge';
 import { MenuIcon } from '@/icons/menu';
 import { ClockIcon } from '@/icons/clock';
+import { ClockIcon } from '@/icons/clock';
```
Comment (pqoqubbw) [inline icons/index.ts:47]: duplication 👀

Comment (shridmishra) [reply icons/index.ts]: deleted it , sorry.

Comment (pqoqubbw) [reply icons/index.ts]: thanks!

**Review by pqoqubbw — APPROVED** [review]

**Pull request merged**